compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -4694,6 +4701,9 @@ private STNode parseFunctionQualifiers(ParserRuleContext context, List<STNode> q
                 case RESOURCE_KEYWORD:
                     qualifier = parseResourceKeyword();
                     break;
+                case ISOLATED_KEYWORD:
+                    qualifier = parseIsolatedKeyword();
```

Comment:
Since we r seeing isolated keyword we can directly consume it. So we can use `consume()` here. Other places will also be changed to consume() in future :-)

Reply:
Ack, fixed just this for now with becd1b0, thanks!